### PR TITLE
Add a session schedule for the KubeVirt Summit

### PR DESCRIPTION
**Pull request by codificat**
**What this PR does / why we need it**:

Add the session schedule for the KubeVirt Summit.

**Does this PR fix any issue?** _(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)_:

> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by codificat** [comment]
/hold
until all reviews/confirmations are in

**Comment by mazzystr** [comment]
@codificat  link checker failing

**Comment by codificat** [comment]
/test kubevirt-io-presubmit-link-checker
apparently there was a DNS glitch

**Review by davidvossel — APPROVED** [review]
/lgtm

File: pages/summit.md [hunk 1]
```diff
@@ -35,7 +32,29 @@ There is no cost for registration. A Linux Foundation account is required.
...
+> note "Note"
+> The times below are all in **UTC**.
+
+This is the session schedule:
+
+| Date   | Time  | Session Title                                                                                                                                                                                         |
+| ------ | ----- | -------------                                                                                                                                                                                         |
+| 9/Feb  | 14:00 | [Automated migration of VMs from VMware or OpenStack to KubeVirt](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/migrating-vms-to-kubevirt-with-coriolis.md) |
+|        | 15:00 | [How to avoid merging broken code with automated testing using prow](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/prow-jobs-automated-testing.md)          |
+|        | 16:00 | [Automating KubeVirt with Tekton Pipelines](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/automating-kubevirt-with-tekton-pipelines.md)                     |
```
Comment (davidvossel) [inline pages/summit.md:44]: It was requested that these sessions and the other 1hr long sessions be reduced to 20 minutes with 5 minutes for questions.

**Review by davidvossel — COMMENTED** [review]
/hold

@codificat we need to resolve whether we're moving ahead with all 30m long sessions or allowing some 1hr long sessions. I already communicated as part of the acceptance that we need everyone to fit within the 30min windows.

File: pages/summit.md [hunk 2]
```diff
@@ -35,7 +32,29 @@ There is no cost for registration. A Linux Foundation account is required.
...
+| 10/Feb | 14:00 | [Moving oVirt and VMware VMs to KubeVirt with VM Import Operator and Forklift](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/kubevirt-vmio.md)              |
+|        | 15:00 | [Privilege dropping, one capability at a time](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/privilege-dropping.md)                                         |
+|        | 16:00 | [Introducing the new KubeVirt driver for Ansible Molecule](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/molecule-kubevirt.md)                              |
+|        | 16:30 | [Virtual Machine Batch API](https://github.com/kubevirt/community/blob/master/events/2021-kubevirt-summit/proposals/vm-batch-api.md)                                                                  |
+|        | 17:00 | The Road to Version 1                                                                                                                                                                                 |
```
Comment (rthallisey) [inline pages/summit.md:54]: Missing link.  Or was there never a link?

Comment (codificat) [reply pages/summit.md:54]: There wasn't, for this and the office hours sessions... the idea is that these two are of "general interest" for the contributors and users track respectively.

I will upload proposals for them so we can link.

File: pages/summit.md (hunk 1 above)
Comment (codificat) [reply pages/summit.md]: Ack. All sessions adjusted to 30 min now, which allowed 3 more sessions to be scheduled.

File: pages/summit.md (hunk 2 above)
Comment (codificat) [reply pages/summit.md]: I updated this PR with a link that will start working once kubevirt/community#78 is merged

**Comment by codificat** [comment]
A last minute change came in, let's
/hold
just a bit more while it gets reviewed

**Comment by codificat** [comment]
Swapped day for one of the sessions. The session itself (HCO) will change slightly (different presenter) but the schedule should be fine now.
/hold cancel

**Review by davidvossel — APPROVED** [review]
/lgtm

**Pull request merged**